python/pyiceberg/table/metadata.py

```diff
@@ -196,28 +197,6 @@ class TableMetadataV1(TableMetadataCommonFields, IcebergBaseModel):
     # because bumping the version should be an explicit operation that is up
     # to the owner of the table.
 
-    @root_validator(pre=True)
```

Comment:
Removing the pre-validators because they aren't very convenient. We have to check everything that goes into it, before any aliases are applied, which makes them messy.